### PR TITLE
added myself to facts.py

### PR DESCRIPTION
**Pull request by vavilovm**
hello?

### CODE REVIEW

File: practice02/facts.py [hunk 1]
```diff
@@ -7,6 +7,9 @@
 people = [
     Person(github_login='example', first_name='Name', last_name='Surname',
            facts=['Fact-right.', 'Fact-write.', 'Fact-right.']),
+    Person(github_login='vavilovm', first_name='mark', last_name='vavilov',
+           facts=['True.', 'False', 'Fact-right.']),
```
Comment (23jura23) [inline practice02/facts.py]: Missed dot after False

File: practice02/facts.py [hunk 2]
```diff
@@ -7,6 +7,9 @@
 people = [
     Person(github_login='example', first_name='Name', last_name='Surname',
            facts=['Fact-right.', 'Fact-write.', 'Fact-right.']),
+    Person(github_login='vavilovm', first_name='mark', last_name='vavilov',
```
Comment (DT6A) [inline practice02/facts.py]: NoT CaPiTaL lEtTeRs

File: practice02/facts.py (hunk 1 above)
Comment (DT6A) [inline practice02/facts.py]: right not left

**Review by DT6A — COMMENTED** [review]
Hello there

**Review by 23jura23 — CHANGES_REQUESTED** [review]
G00d 0r y0u

**Pull request closed without merging**